### DISCUSSION TITLE
Defining periodic pairs using gmsg2nek

### DISCUSSION THREAD

**Discussion by sinatahmooresi** [Nek5000/nekRS#491] [Meshing]
I am trying to  use gmsh2nek to convert my circular pipe built by gmsh to nek. I get this error: 
![gsm2nek](https://user-images.githubusercontent.com/43316204/233849190-1abe3c20-8f86-4b50-b22e-0b557d72265e.JPG)

**Comment by stgeke** [comment]
Please use the `nek5000` mailing list for request like this.
